<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>congee</class>
 <widget class="QDialog" name="congee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
    color: #000;
    border: none;
}
employeeUI {
    background-color: #eff99fe;
}
#leftmenu {
    background-color: #2596be;
}
#label_3 {
    background-color: #2596be;
}
#pushButton_16{
    background-color: #2596be;
}
#pushButton_17{
    background-color: #2596be;
}
#pushButton_18{
    background-color: #2596be;
}
#pushButton_19{
    background-color: #2596be;
}
#pushButton_20{
    background-color: #2596be;
}
#pushButton_21{
    background-color: #2596be;
}
#pushButton_22{
    background-color: #2596be;
}
#frame_10{
    background-color: #2596be;
}
#frame_9 {
    background-color: #2596be;
}
#frame_11 {
    background-color: #2596be;
}
QLineEdit {
    background: transparent;
}
QFrame {
    border-radius: 10px;
    border: 2px solid #2596be;
}
QLabel {
    color: #2596be;
}
#widget {
    background-color: #cecece;
}
#lineEdit_2 {
    background: transparent;
}
#label_3 {
    color: #000;
    border: #000;
}
#lineEdit_3 {
    background: #ffffff;
    background: transparent;
}
#datedebut{
    background: #ffffff;
    background: transparent;
}
#lineEdit_3 {
    background: #ffffff;
    background: transparent;
}
#label_4 {
    color: #000;
    border: #000;
}
#id {
    background: #ffffff;
}
#datedebut{
    background: #ffffff;
}
#datefin{
    background: #ffffff;
}
#label_5 {
    color: #000;
    border: #000;
}
#nom {
    background: #ffffff;
}
#label_6 {
    color: #000;
    border: #000;
}
#prenom {
    background: #ffffff;
}
#label_7 {
    color: #000;
    border: #000;
}
#post {
    background: #ffffff;
}
#label_8 {
    color: #000;
    border: #000;
}
#sexe{
    background: #ffffff;
}
#label_9 {
    color: #000;
    border: #000;
}
#salaire {
    background: #ffffff;
}
#label_10 {
    color: #000;
    border: #000;
}
#ajouter {
    border-radius: 10px;
    border: 3px solid #2596be;
}
#pushButton_4 {
    border-radius: 10px;
    border: 3px solid #2596be;
}
#editer {
    border-radius: 10px;
    border: 3px solid #2596be;
}
#pushButton_6 {
    border-radius: 10px;
    border: 3px solid #2596be;
}
#label_11 {
    border: #000;
}
#label_12 {
    border: #000;
}
#label_23 {
    border: #000;
}
#frame_4 {
    border-radius: 10px;
    border: 1px solid #2596be;
}
#lineEdit_10 {
    background: #cecece;
}
#widget_3 {
    background-color: #fefeff;
}
#mainbody {
    background-color: #ffffff;
}
#frame_8 {
    border-radius: 10px;
    border: 1px solid #2596be;
}
#tableView1 {
    background-color: #2596be;
}
#lineEdit_supp {
    border-radius: 6px;
    border: 1px solid #4cd814;
    background: #ffffff;
}
#supp {
    border-radius: 10px;
    border: 2px solid #090909;
    background: #fffff;
}
#ExportationButton {
    border-radius: 20px;
    border: 2px solid #4cd814;
    background: #ffffff;
}
#statistiqueButton {
    border-radius: 20px;
    border: 2px solid #2596be;
    background: #ffffff;
}
#conge {
    border-radius: 7px;
    border: 2px solid #000;
    background: #ffffff;
}
#label_24 {
    color: #000;
    border: #000;
}
#label_25 {
    color: #000;
    border: #000;
}
#label_26 {
    color: #000;
    border: #000;
}
#logo {
    border-radius: 6px;
    border: 3px solid #4cd814;
    background: #ffffff;
}
#sous {
    background-color: #cecece;
}
#label_27 {
    border-radius: 1px;
    border: 1px solid #ffffff;
    color: #000;
}
#label_28 {
    border-radius: 1px;
    border: 1px solid #ffffff;
    color: #000;
}
#label_29 {
    border-radius: 1px;
    border: 1px solid #ffffff;
    color: #000;
}
#label_30 {
    border-radius: 1px;
    border: 1px solid #ffffff;
    color: #000;
}
#label_31 {
    border-radius: 1px;
    border: 1px solid #ffffff;
    color: #000;
}
#label_32 {
    border-radius: 1px;
    border: 1px solid #ffffff;
    color: #000;
}
#pushButton_17 {
    background-color: #fefeff;
    padding: 5px;
    text-align: left;
    border-top-left-radius: 20px;
}
#pushButton_22 {
    color: #4cd814;
}
#comboBox_critere {
    border-radius: 10px;
    border: 2px solid transparent;
}
#comboBox_2 {
    border-radius: 10px;
    border: 2px solid transparent;
}
#label {
 	color: #000;
   border: #000;
   background: transparent;
}
#lineEdit_10 {
    background: transparent;
}
#validerButton{
    border-radius: 10px;
    border: 3px solid #2596be;
	background:  #cecece;
}
#label_5,#label_6,#label_7,#label_8
{
	color: #2596be;
}
#pushButton {
    border-radius: 7px;
    border: 2px;
    background: #ffffff;
}
#pushButton_4 {
    border-radius: 10px;
    border: 3px solid #2596be;
}</string>
  </property>
  <widget class="QWidget" name="leftmenu" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>565</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_9">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame_11">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <widget class="QFrame" name="frame_10">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>160</width>
            <height>61</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <widget class="QLabel" name="logo">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>10</y>
             <width>51</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image22.qrc">:/res/image22</pixmap>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>15</y>
             <width>101</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>MedicaPRO</string>
           </property>
          </widget>
         </widget>
         <widget class="QPushButton" name="pushButton_16">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>70</y>
            <width>75</width>
            <height>24</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Menu</string>
          </property>
          <property name="icon">
           <iconset resource="image23.qrc">
            <normaloff>:/res/icone23</normaloff>:/res/icone23</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_17">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>120</y>
            <width>161</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <italic>true</italic>
            <bold>true</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="text">
           <string>Employé</string>
          </property>
          <property name="icon">
           <iconset resource="image27.qrc">
            <normaloff>:/res/icone27</normaloff>:/res/icone27</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_18">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>141</width>
            <height>24</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <italic>true</italic>
            <bold>true</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="text">
           <string>Médicaments</string>
          </property>
          <property name="icon">
           <iconset resource="image26.qrc">
            <normaloff>:/res/icone26</normaloff>:/res/icone26</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_19">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>220</y>
            <width>141</width>
            <height>24</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Fournisseurs</string>
          </property>
          <property name="icon">
           <iconset resource="image25.qrc">
            <normaloff>:/res/icone25</normaloff>:/res/icone25</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_20">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>270</y>
            <width>141</width>
            <height>24</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Ordonnances</string>
          </property>
          <property name="icon">
           <iconset resource="image23.qrc">
            <normaloff>:/res/icone23</normaloff>:/res/icone23</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_21">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>330</y>
            <width>121</width>
            <height>24</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Financières</string>
          </property>
          <property name="icon">
           <iconset resource="image24.qrc">
            <normaloff>:/res/icone24</normaloff>:/res/icone24</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_22">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>470</y>
            <width>151</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Se déconnecter</string>
          </property>
          <property name="icon">
           <iconset resource="image30.qrc">
            <normaloff>:/res/icone30</normaloff>:/res/icone30</iconset>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="mainbody" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>701</width>
     <height>565</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>135</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignRight">
      <widget class="QWidget" name="widget_7" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="spacing">
         <number>7</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="image.qrc">
            <normaloff>:/res/icone1.jog.png</normaloff>:/res/icone1.jog.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="image2.qrc">:/res/icone7</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_8" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>553</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="image.qrc">:/res/icone2.jpg.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Recherche</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_9" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <property name="spacing">
         <number>9</number>
        </property>
        <property name="leftMargin">
         <number>50</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item alignment="Qt::AlignRight">
         <widget class="QPushButton" name="pushButton_4">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTipDuration">
           <number>-2</number>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="image.qrc">
            <normaloff>:/res/icone4.jpg.png</normaloff>:/res/icone4.jpg.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_15">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="image20.qrc">
            <normaloff>:/res/icone20</normaloff>:/res/icone20</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>51</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image16.qrc">:/res/icone16</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>49</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image18.qrc">:/res/icone18</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>681</width>
      <height>191</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
    <widget class="QComboBox" name="type">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Annuel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Paternite/Maternite</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Maladie</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Date Départ</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Date Retour</string>
     </property>
    </widget>
    <widget class="QPushButton" name="validerButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Envoyer la demande</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateDebut">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateFin">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>60</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>451</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Mes demandes</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image20.qrc"/>
  <include location="image16.qrc"/>
  <include location="image2.qrc"/>
  <include location="image24.qrc"/>
  <include location="image22.qrc"/>
  <include location="image26.qrc"/>
  <include location="image25.qrc"/>
  <include location="image27.qrc"/>
  <include location="image18.qrc"/>
  <include location="image30.qrc"/>
  <include location="image23.qrc"/>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
